studio/components/interfaces/Integrations/OrganizationPicker.tsx

```diff
@@ -63,21 +70,22 @@ const OrganizationPicker = ({
             size="medium"
             block
             className="justify-start"
-            icon={<IconHexagon />}
             loading={isLoadingOrganization}
-            disabled={isLoadingOrganization}
+            disabled={disabled}
```

Comment:
do we not want to disable the button as well while the organizations is loading?

Reply:
ah okay i can see that's the behaviour from integrations/vercel, but integrations github isnt passing a disabled prop - just to check here

Reply:
yeap no worries, just address the comments which you think are more important